File: Quiz.Android/lib/models/question.dart
import 'package:quiz/models/option.dart';

class Question {
  final int questionId;
  final String title;
  final bool isActive;
  final List<Option> options;
  final Option? selected;

  const Question({
    required this.questionId,
    required this.title,
    required this.isActive,
    required this.options,
    this.selected,
  });

  factory Question.fromJson(Map<String, dynamic> json) {
    return Question(
      questionId: json['questionId'],
      title: json['title'],
      isActive: json['isActive'],
      options:
          (json["options"] as List).map((x) => Option.fromJson(x)).toList(),
    );
  }
}

File: Quiz.Entity/Models/Question.cs
using System.ComponentModel.DataAnnotations;

namespace Quiz.Entity.Models
{
    public class Question
    {
        [Key]
        public int QuestionId { get; set; }
        public string Title { get; set; }
        public bool IsActive { get; set; }
        public string Description { get; set; }

        public virtual ICollection<Option> Options { get; set; }
    }
}